protected/humhub/modules/user/models/fieldtype/CountrySelect.php

```diff
@@ -55,22 +55,22 @@ public function getSelectItems()
 
         // if no options set basically return a translated map of all defined countries
         if (empty($this->options) || trim($this->options) == false) {
-            $items = iso3166Codes::$countries;
-            foreach ($items as $code => $value) {
-                $items[$code] = iso3166Codes::country($code);
+            $isoCodes = Iso3166Codes::$countries;
+
+            foreach ($isoCodes as $code => $value) {
+                $items[Iso3166Codes::country($code)] = Iso3166Codes::country($code);
             }
         } else {
             foreach (explode(",", $this->options) as $code) {
-
                 $key = trim($code);
-                $value = iso3166Codes::country($key, true);
+                $value = Iso3166Codes::country($key, true);
                 if (!empty($key) && $key !== $value) {
                     $items[trim($key)] = trim($value);
```

Comment:
I'm not sure why `$key` is being `trim()` here of `$key` equals `$code` which is already `trim()` :thinking:

Reply:
If the variable `$key` is already equal to a trimmed version of another variable, there is no need to trim it again when assigning it to the `$items` array.  If `$key` is obtained through `$key = trim($code)`, and later you check if it's not empty and different from `$value`, you can directly use `$key` without trimming it again when assigning it to `$items`.

If the variable is already equal to a trimmed version, you don't need to trim it again when using it in subsequent operations unless there are specific reasons or requirements to do so?